Tools/baselineDef.cc

```diff
@@ -1028,22 +1247,19 @@ bool BaselineVessel::GetMHT() const
   // Calculate MHT
   TLorentzVector MHT(0, 0, 0, 0);
   float SumHT = 0.0; //Using jet > 30 , |eta| < 5
-  for(auto &jet : tr->getVec<TLorentzVector>("jetsLVecLepCleaned"))
+  for(auto &jet : tr->getVec<TLorentzVector>("prodJetsNoLep_jetsLVec"))
```

Comment:
Do we always want lep cleaned here., or should there be a switch here?

Reply:
Yes, it should be wrap by UseNoLepVar(). Will update